parsl/monitoring/monitoring.py

```diff
@@ -429,6 +429,7 @@ def start(self,
 
                         msg_0: AddressedMonitoringMessage
                         msg_0 = (msg, 0)
+                        unknown_type_mg = f"Discarding message from interchange with unknown type {msg[0].value}"
```

Comment:
It would be better style here to format the message only if it is needed - for example right before the error message is logged around line 452

Reply:
I had a look at this and since msg_0 is initialized at line 475 the best possible place for me to initialize the error message was line 476. I looked at another possible solution and realized that breaking the line and having the '# type: ignore[unreachable]' comment on the first line of the broken sentence solves the unreachable issue I previously had. I don't know why I had not thought of this before